explorer/app/models/responses.ts

```diff
@@ -95,10 +123,15 @@ export interface FileListResponse extends PaginatedResponse {
   hits: FileResponse[];
 }
 
+export interface AnvilFileListResponse extends PaginatedResponse {
```

Comment:
It looks like `AnvilFileListResponse` is not referenced anywhere?

Reply:
you're right. Rethink about the `ListResponseType` and `DetailResponseType`, I don't think both of these types make sense anymore. Since configuration accepts different types, we don't know them beforehand. 
So I turn all these ListResponse into a since interface and removed `ListResponseType` and `DetailResponseType` types.
Let me know if that makes sense for you